app/packages/state/src/recoil/schema.ts

```diff
@@ -329,9 +329,24 @@ export const labelFields = selectorFamily<string[], { space?: State.SPACE }>({
     ({ get }) => {
       const paths = get(fieldPaths(params));
 
-      return paths.filter((path) =>
+      const flatLabelFields = paths.filter((path) =>
         LABELS.includes(get(field(path)).embeddedDocType)
       );
+
+      const dynamicLabelFields = paths
+        .filter(
+          (path) =>
+            get(field(path)).embeddedDocType ===
+            "fiftyone.core.odm.embedded_document.DynamicEmbeddedDocument"
```

Comment:
I recommend including this as a constant in `@fiftyone/utilities`. This filtering could also be baked into the `fieldPaths` and `fields` selectors but I can look into that